library/src/com/google/maps/android/geojson/GeoJsonLineStringStyle.java

```diff
@@ -49,6 +49,15 @@ public void setColor(int color) {
         mPolylineOptions.color(color);
         styleChanged();
     }
+    
+    /**
+     * Sets the GeoJsonLineString clickable
```

Comment:
The JavaDoc on the others has been copied from the Android Maps API. Please do the same here - that is:
> Specifies whether this polyline is clickable.

Also remove the empty line below.